front/internals/webpack/webpack.config.js

```diff
@@ -1,37 +1,48 @@
 const path = require('path');
+
```

Comment:
A lot of the changes in this file are due to the fact that it was excluded from prettier for some reason. I included it but the diff might be a bit confusing now. Apologies!